### PR TITLE
Initial release

### PR DESCRIPTION
**Pull request by chris-martin**
- Added more stackage resolvers; not going back past ghc 9.2 because NoFieldSelectors, OverloadedRecordDot, ImportPostQualified.
- Fixed a bug; had the name of the `endLine` property wrong, noticed just by perusing the output in the test assertion which was also written wrong.
- Converted readme to lhs, added a few documentation examples.
- Release as 0.0.0.0

### CODE REVIEW

File: package.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: github-workflow-commands
 version: 0.0.0.0
 maintainer: Freckle Education
-category: Utils
+category: GitHub
```
Comment (pbrisbin) [inline package.yaml:4]: Wow this is really [the wild west](https://hackage.haskell.org/packages/) huh?

File: weeder.toml
Comment (pbrisbin) [inline weeder.toml:1]: I think it was/is a mistake to include the weeder setup in libraries. Weeder is good for applications, where everything not reachable from `main` is dead code. In a library exposing public API to outside consumers, you have no idea what's unused and weeder's heuristic is not applicable.

**Review by pbrisbin — APPROVED** [review]

File: weeder.toml
Comment (chris-martin) [reply weeder.toml:1]: I'm on the fence; in libraries if you have the spec as a weeder root then it sort of serves as a coverage check at least?

File: package.yaml (hunk 1 above)
Comment (chris-martin) [reply package.yaml:4]: Yeah I try to find some balance between "is this category too broad to be useful" or "is this category so narrow it's just clutter." I still find that page listing packages by category to be helpful every once in a while when I'm looking for something, but the Foundation is going to have to apply some sort of top-down organization effort if they ever want it to actually be good.

**Pull request merged**